### ISSUE TITLE
Hunt script requests

### ISSUE THREAD

**Issue by Obscillesk** [nsweeting2/XaosUI#1]
Web killer (though I realized we could probably fix  this by adding area keywords to the webs within the area)

A customizable delay for slowly pushing into a strong area, or a place that likes to accumulate more and more shit hunting. (maze)

**Comment by nsweeting2** [comment]
'scripterRate #' can be used to slow the hunt pace. Or are you looking for something that goes slow then ramps up to the default rate over time?

**Comment by Obscillesk** [comment]
Nah, that's fine. Like conventional wisdom with March's scripts is that you need to add a 2-3 second delay on the hunt (which he has accessible in a drop down [which is pretty obnoxious to use]) if you're having trouble with them piling up on you

**Comment by nsweeting2** [comment]
So scripterRate works for to accomplish this?

**Comment by nsweeting2** [comment]
Added a web killer for QoL, its useful even in the event npc's get area kill keywords. Available next update.

**Comment by nsweeting2** [comment]
Jumped the gun, Need to know if scripterRate covers the other side of the request.
